zerver/lib/test_helpers.py

```diff
@@ -598,3 +599,31 @@ def zulip_reaction_info() -> Dict[str, str]:
         emoji_code='zulip',
         reaction_type='zulip_extra_emoji',
     )
+
+@contextmanager
+def mock_queue_publish(
+        at: str,
```

Comment:
What does `at` stand for?

Reply:
Where `queue_json_publish` was imported at.  I'll swap it out for `method_to_patch`.